src/ddl/install_scratch.sql

```diff
@@ -35,8 +35,8 @@ CREATE TABLE flow_diagrams (
         GENERATED BY DEFAULT AS IDENTITY ( START WITH 1 NOCACHE )
     NOT NULL,
     dgrm_name     VARCHAR2(150 CHAR) NOT NULL,
-    dgrm_version  VARCHAR2(10 CHAR),
-    dgrm_status   VARCHAR2(10 CHAR),
+    dgrm_version  VARCHAR2(10 CHAR) DEFAULT '0' NOT NULL,
+    dgrm_status   VARCHAR2(10 CHAR) DEFAULT 'draft' NOT NULL,
     dgrm_category VARCHAR2(30 CHAR),
     dgrm_last_update TIMESTAMP WITH TIME ZONE DEFAULT systimestamp NOT NULL,
```

Comment:
Question:
Do we always use timestamp with timezone or always without?

Reply:
I was thinking that maybe we could just use date for diagram changes and meta-process level things, but should use timestamp for process runtime date fields? (We use DATE for the subflow_log - which should be a timestamp!)  We use timestamp with timezone everywhere else.  So I'd go for either just Date or 'with timezone' like everywhere else.